README.md

```diff
@@ -56,7 +56,7 @@ macOS: `brew install x264`
 
 Linux: `libopus-dev libopusfile-dev`
 
-macOS: `brew install opus`
+macOS: `brew install opus opusfile`
```

Comment:
What's this for?

Reply:
When I was setting up for development I was getting an error a lot like the one you fixed here: https://github.com/viamrobotics/gostream/commit/4b750ece3102daf76a921ede71c78ee509a2e491

I thought it would be helpful to include the parallel package for Mac in the list of prerequisites, because it took me a minute to figure out what my missing dependency was

Reply:
Oh, nice. Thanks for the improvement! LGTM.